pegasus/sites.v3/code.org/views/carousel_blockchain_video.haml

```diff
@@ -0,0 +1,25 @@
+- video_blockchain_index = 7
+- video_blockchain_caption = [hoc_s(:blockchain_video_title_01), hoc_s(:blockchain_video_title_02), hoc_s(:blockchain_video_title_03), hoc_s(:blockchain_video_title_04), hoc_s(:blockchain_video_title_05), hoc_s(:blockchain_video_title_06), hoc_s(:blockchain_video_title_07)]
+- video_blockchain_id = ["introduction_blockchain", "why_blockchain", "under_hood", "beyond_currencies", "market_price", "societal_impact", "trustworthy_scam"]
+- video_blockchain = ["MjibPL4tGqo", "10wYLCnDHao", "It_Ivgm5wQA", "3EwkfUz_BPs", "d9rKkk3oVdY", "WzxbwEQRJPo",  "C3C_JfG8cNg"]
+
+.carousel-wrapper#blockchain_works_collection
+  .carousel.blockchain__works__collection
+    .slides
+      - video_blockchain_index.times do |index|
+        .slide
+          .action-block.action-block--one-col.flex-space-between
+            .video__container
+              = view :display_video_thumbnail, id: video_blockchain_id[index], video_code: video_blockchain[index], play_button: 'center', letterbox: 'false', caption: video_blockchain_caption[index]
+            
+  .pagination
+    #blockchain_works_collection_pagination
+  .dd__buttons
+    %button.carousel__prev#prev_5
+      %img{src: "/images/carousel/prev.svg", alt: "previous"}
+
+    %button.carousel__next#next_5
+      %img{src: "/images/carousel/next.svg", alt: "next"}
+
+
+
```

Comment:
Nit: extra whitespace — there can just be one line of whitespace at the end of the file